ios/browser/brave_web_client.mm

```diff
@@ -33,9 +33,23 @@
 }
 
 std::string BraveWebClient::GetUserAgent(web::UserAgentType type) const {
+  if (user_agent_.empty()) {
+    return ChromeWebClient::GetUserAgent(type);
+  }
   return user_agent_;
 }
 
+void BraveWebClient::AddAdditionalSchemes(Schemes* schemes) const {
+  ChromeWebClient::AddAdditionalSchemes(schemes);
+
+  schemes->standard_schemes.push_back(kBraveUIScheme);
```

Comment:
these should also be in `cors_enabled_schemes` and `savable_schemes` to match `BraveContentClient::AddAdditionalSchemes` and also to match handling of `kChromeUIScheme` in `RegisterContentSchemes`

@diracdeltas it looks like `kBraveUIScheme` should also be added to `service_worker_schemes` in desktop per `RegisterContentSchemes`, but not sure if that applies to ios or not. Maybe the best thing here in general is to check each list for `kChromeUIScheme` and add `kBraveUIScheme` if it's present? We could probably add a helper that could be used in desktop and ios

Reply:
```
// This should only be registered if the
  // kEnableServiceWorkerForChrome or
  // kEnableServiceWorkerForChromeUntrusted feature is enabled but checking it
  // here causes a crash when --no-sandbox is enabled. See crbug.com/1313812
  // There are other render side checks and browser side checks that ensure
  // service workers don't work for chrome[-untrusted]:// when the flag is not
  // enabled.
```
but I don't see any `kEnableServiceWorkerForChrome` feature

Reply:
@bridiver iOS only contains standard + secure schemes: https://source.chromium.org/chromium/chromium/src/+/main:ios/web/public/web_client.h;l=68;drc=c85f08af47c25bc5eb4a7f81c0ac0620d8d767ca;bpv=0;bpt=1

Reply:
>@bridiver iOS only contains standard + secure schemes: https://source.chromium.org/chromium/chromium/src/+/main:ios/web/public/web_client.h;l=68;drc=c85f08af47c25bc5eb4a7f81c0ac0620d8d767ca;bpv=0;bpt=1

The comment says that `standard schemes` are those that follow RFC3986 syntax, which should include `brave://` and `chrome://`, no?